### DISCUSSION TITLE
Can't create an issue draft from a github project view

### DISCUSSION THREAD

**Discussion by furagi** [community/community#116547] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

In one of our organization's projects two of our QA engineers are experiencing what seems to me a bug - they can't create an item (an issue draft) from a table view (the view has filters `no:sprint is:issue is:open `). You can check the attached video. Neither Enter key nor clicking on the + icon don't work (btw, I'm able to add an item using the Enter key, but also can't using the + icon, if the field isn't empty). There's nothing in the developer console, the web app doesn't submit any API requests. The project items count is way below 1200 and same goes for the archived items. Both of the engineers belong to the same team that has write access to the project. Also, the team consists of 3 members (the org admin is the 4th one), and the 3rd one, who is not an org admin, can create drafts. One more thing is that both of the QA engineers having the issue haven't experienced it 1-2 weeks ago. Neither incognito mode, nor using another browser (the issue occurs in both Mozilla Firefox and Google Chrome) didn't help.

The url used to add an item to the board: https://github.com/orgs/bazaarorigin/projects/6/views/1
So, the conditions (it's not exact steps to reproduce because as I already said the 3rd member of the team doesn't face it):

- create a Github project for an organization
- add a table view with filters `no:sprint is:issue is:open` (`sprint` is  an iteration field)
- add to the organization a team with write access to the project
- try to add to the table view an item from the table view

https://github.com/community/community/assets/5179046/8f1d7f0b-f9dc-4b76-a5cf-5a365e9318f0

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by KISGP** [comment]
Me too.Do you know how to fix it now? @furagi
